### DISCUSSION TITLE
Transactions not updating

### DISCUSSION THREAD

**Discussion by NerdPlayground** [DevSkillsHQ/discusssions#73] [Help] [answered]
I can list all the available transactions but no transaction I create is being added to the list

**Comment by fenske** [comment]
Hi @NerdPlayground. I'm Anton from Alva. Thanks for reporting this issue; I'm sorry to hear you've encountered it! There is an intermittent technical disruption with our cloud provider hosting the underlying DB. I'll look at what's possible to do and update you as soon as I have something to share.

**Comment by fenske** [comment, marked as answer]
@NerdPlayground The issue has now been fixed. You can try using the API again, and it should work. Let me know if you have any questions. I wish you the best of luck with the rest of your interview process!
